tests/unit/autojac/test_mtl_backward.py

```diff
@@ -227,43 +158,13 @@ def test_mtl_backward_various_shared_params(shared_params_shapes: list[tuple[int
     mtl_backward(
         losses=[y1, y2],
         features=representations,
-        tasks_params=[[p1], [p2]],
-        shared_params=shared_params,
         A=UPGrad(),
     )
 
-    for p in [*shared_params, p1, p2]:
+    for p in [*shared_params]:
```

Comment:
I think we still have to test the grads of `p1` and `p2`.

Reply:
Yeah, I think this test is a bit confusing because p1 and p2 are actually never used.

Reply:
I reverted the change and added a comment